.eslintrc

```diff
@@ -14,3 +14,4 @@ rules:
   no-use-before-define: [warn, { functions: false }]
   react/no-multi-comp: [off]
   no-throw-literal: [off]
+  import/no-unresolved: [warn]
```

Comment:
i was getting errors with:
https://github.com/mozilla/morgoth/pull/13/files#diff-535db65efe188b02b6f9792222d76a0bR4

```
Unable to resolve path to module 'redux-form-material-ui'.
```
couldn't get rid of them and it was working as intended so i switched to a warning so i could commit. but maybe there's a way of fixing the error?